### PR TITLE
Release 4.2.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.2.0](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.2.0)

**Review by Blackbaud-AlexKingman — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 4.2.0 (2020-09-25)
+
+- Add the ability to limit the countries available for selection in the phone field component. [#68](https://github.com/blackbaud/skyux-phone-field/pull/68)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Changed "add" to "added" and you're good!

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Fixed

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.2.0](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.2.0)

**Pull request merged**